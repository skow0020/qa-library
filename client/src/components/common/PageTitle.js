import PropTypes from 'prop-types';
import React from 'react';
import Typography from '@material-ui/core/Typography';

export default function PageTitle({ title }) {
  return (
      <Typography gutterBottom className="page-title" variant="h3">
        {title}
      </Typography>
  );
};

PageTitle.propTypes = {
  title: PropTypes.string.isRequired
};