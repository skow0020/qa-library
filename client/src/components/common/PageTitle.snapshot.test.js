import PageTitle from './PageTitle';
import React from 'react';
import renderer from 'react-test-renderer';

describe('PageTitle Snapshot Tests', () => {
  test('PageTitle Snapshot', () => {
    const component = renderer.create(
      <PageTitle title="Add an Article"></PageTitle>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});