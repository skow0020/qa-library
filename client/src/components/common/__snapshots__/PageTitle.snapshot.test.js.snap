// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Snapshot Tests PageTitle Snapshot 1`] = `
<div
  className="text-center text-md-left mb-sm-0 col-12 col-sm-4"
>
  <span
    className="text-uppercase page-subtitle"
  />
  <h3
    className="page-title"
  >
    Add an Article
  </h3>
</div>
`;
